.gitignore

```diff
@@ -110,8 +110,8 @@ celerybeat-schedule
 
 # dotenv
 .env
-.env_staging
```

Comment:
@tchaton I dont think you're supposed to commit your .gitignore?

Reply:
Yes. I aligned the .env file with platform.